plugins/hls-rename-plugin/test/Main.hs

```diff
@@ -21,6 +21,9 @@ tests = testGroup "Rename"
         rename doc (Position 0 15) "Op"
     , goldenWithRename "Exported function" "ExportedFunction" $ \doc ->
         rename doc (Position 2 1) "quux"
+    , ignoreForGhcVersions [GHC90, GHC92] "Record field refs broken for ghc > 9" $
+      goldenWithRename "Field Puns" "FieldPuns" $ \doc ->
+        rename doc (Position 7 13) "bleh"
```

Comment:
What is missing for GHC 9.x?

Reply:
Write it down in the code so we remember! Even better, make a ticket and link to it.

Reply:
I made a [ticket](https://github.com/wz1000/HieDb/issues/45) and linked it in a comment.